.changesets/config_david_subgraph_metrics_ga.md

```diff
@@ -0,0 +1,11 @@
+### [Subgraph Insights] Subgraph metrics config flag now GA ([PR #8200](https://github.com/apollographql/router/pull/8200))
```

Comment:
@david-castaneda FYI this (and the bottom of the file) should be linking #8392 not #8200